example_configs/wildfly-10.yaml

```diff
@@ -0,0 +1,35 @@
+---
+lowercaseOutputName: true
+lowercaseOutputLabelNames: true
+whitelistObjectNames:
```

Comment:
Given this is a framework, you shouldn't assume only the wildfly objects are of potential interest